drivers/bluetooth/hci/spi.c

```diff
@@ -17,9 +17,9 @@
 #include <zephyr/bluetooth/hci.h>
 #include <zephyr/drivers/bluetooth/hci_driver.h>
 
-#define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_DEBUG_HCI_DRIVER)
-#define LOG_MODULE_NAME bt_driver
-#include "common/log.h"
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(bt_driver, CONFIG_BT_HCI_DRIVER_LOG_LEVEL);
```

Comment:
Why is it here ok to pass the Kconfig option directly to `LOG_MODULE_REGISTER()` but not in other places?

Reply:
conditional inclusion of the log template in kconfig:
if the template is if'd out, (e.g. because of `BT_SMP` not being set), the `SMP` log options won't exist and `LOG_MODULE_REGISTER` will not work since the define will not exist.

in this case, the template is not excluded depending on some kconfig, so it works

Reply:
This is a mistake for this case. I took a look again at all the file and fixed the same mistake on 7 others files.

Let me add that some files like `subsys/bluetooth/host/l2cap.c` pass the Kconfig option directly to `LOG_MODULE_REGISTER` because the Kconfig is not depending on Kconfig that could be disabled and so their Kconfig log symbol is always defined.